README.md

```diff
@@ -59,7 +59,7 @@ The directory where your Brewfile is located.
 
 ## Dependencies
 
-None.
+* [elliotweiser.osx-command-line-tools](https://galaxy.ansible.com/elliotweiser/osx-command-line-tools/)
```

Comment:
Just to clarify, there is no missing commit. This change already ensures that the CLTs are installed because the homebrew role executes its dependency prior to executing its own tasks.

Also, I found the SLAs:

http://images.apple.com/legal/sla/docs/macOS1012.pdf
http://images.apple.com/legal/sla/docs/OSX1011.pdf
http://images.apple.com/legal/sla/docs/OSX10103.pdf

Section 2.B.iii confirms what I said.